taskRequests/script.js

```diff
@@ -84,6 +93,173 @@ function openTaskDetails(id) {
   window.location.href = url;
 }
 
+if (isDev) {
+  containerFilter.style.display = 'flex';
+  oldContainerFilter.style.display = 'none';
+} else {
+  containerFilter.style.display = 'none';
+  oldContainerFilter.style.display = 'flex';
+}
+sortButton.addEventListener('click', async (event) => {
+  event.stopPropagation();
+  sortModal.classList.toggle('hidden');
+  backDrop.style.display = 'flex';
+  sortModalButtons();
+});
+
+backDrop.addEventListener('click', () => {
+  sortModal.classList.add('hidden');
+  filterModal.classList.add('hidden');
+  backDrop.style.display = 'none';
+});
+
+filterModal.addEventListener('click', (event) => {
+  event.stopPropagation();
+});
+
+sortModal.addEventListener('click', (event) => {
+  event.stopPropagation();
+});
+
+filterButton.addEventListener('click', (event) => {
+  filterModal.classList.toggle('hidden');
+  backDrop.style.display = 'flex';
+  event.stopPropagation();
```

Comment:
Is this required? as you already have the same in line [117](https://github.com/Real-Dev-Squad/website-dashboard/pull/576/files#diff-23b61ff1789eab0a1640feae99a0bf3465052f6c0cdda786aa4ae08c876715b5R117)

Reply:
fixed